alpine/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM alpine:3.5
-ENV CLOUD_SDK_VERSION 160.0.0
+ENV CLOUD_SDK_VERSION 161.0.0
 ENV PATH /google-cloud-sdk/bin:$PATH
-RUN apk --no-cache add curl python py-crcmod bash libc6-compat && \
+RUN apk --no-cache add curl python py-crcmod bash libc6-compat openssh git && \
```

Comment:
can you replace this with `openssh-client` please:

https://pkgs.alpinelinux.org/packages?name=openssh-client&branch=&repo=&arch=&maintainer=